GalaxyZooTouchTable/GalaxyZooTouchTable/Models/TableUserFactory.cs

```diff
@@ -13,11 +13,11 @@ enum UserType
         Heart
     }
 
-    abstract class TableUser
+    public class TableUser
```

Comment:
This was changed because I'm thinking of using an `ObservableCollection<TableUser>` group in a future PR to track when users come to the table. This will help the main app level know who is at the table for purposes of asking X user for help with a classification and knowing what to display on the center of the app. 

However, using `ObservableCollection<TableUser>` with an abstract is not allowed due to accessibility levels of an abstract class.

Reply:
Would it work if TableUser was an interface or was private? In order to enforce using the factory class you made, I think it's important that you can't just instantiate a table user without any of the predefined values

Reply:
I'll try one of those options. I definitely agree that is a better option.